### ISSUE TITLE
Searching for a finding ID throws error

### ISSUE THREAD

**Issue by madchap** [DefectDojo/django-DefectDojo#2480]
**Bug description**

In any case, the search is not conclusive when looking for a specific ID, whether directly or with `finding:<finding_id>`. The result is always blank.

**When being a regular "active" user (e.g. not "staff").**

![image](https://user-images.githubusercontent.com/5468769/83753102-f4610f80-a669-11ea-8b80-ae586a8caf5f.png)

```
Traceback:

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "./dojo/search/views.py" in simple_search
  101.                                 request.user]),))

File "/usr/local/lib/python3.5/site-packages/django/db/models/manager.py" in manager_method
  82.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/db/models/query.py" in filter
  892.         return self._filter_or_exclude(False, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/db/models/query.py" in _filter_or_exclude
  910.             clone.query.add_q(Q(*args, **kwargs))

File "/usr/local/lib/python3.5/site-packages/django/db/models/sql/query.py" in add_q
  1290.         clause, _ = self._add_q(q_object, self.used_aliases)

File "/usr/local/lib/python3.5/site-packages/django/db/models/sql/query.py" in _add_q
  1318.                     split_subq=split_subq, simple_col=simple_col,

File "/usr/local/lib/python3.5/site-packages/django/db/models/sql/query.py" in build_filter
  1190.         lookups, parts, reffed_expression = self.solve_lookup_type(arg)

File "/usr/local/lib/python3.5/site-packages/django/db/models/sql/query.py" in solve_lookup_type
  1049.         _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())

File "/usr/local/lib/python3.5/site-packages/django/db/models/sql/query.py" in names_to_path
  1420.                                      "Choices are: %s" % (name, ", ".join(available)))

Exception Type: FieldError at /simple_search
Exception Value: Cannot resolve keyword 'authorized_users' into field. Choices are: cve, cwe, description, id, impact, last_used, mitigation, numerical_severity, references, severity, tagged_items, template_match, template_match_title, title
```

**Steps to reproduce**
1. Type in a finding ID in the simple search box
2. Experience the error

**Expected results**
- The search returns one result: the actual finding
- There is no error.

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [x] Docker
- [ ] setup.bash / legacy-setup.bash

**Comment by valentijnscholten** [comment]
Error should be fixed by the PR. Searching by (finding) id is currently not possible and by the way watson currently indexes and returns its results I'm not sure if it can be added easily.
If only we would have some time to add something like elastic search or an embedded lucene/solr.

**Comment by valentijnscholten** [comment]
closing as the fix is merged into 1.6.1. Will look at adding "search by finding id" later after discussing with @madchap

**Comment by madchap** [comment]
Thanks @valentijnscholten :100:
